src/zebras/_timeframe.py

```diff
@@ -805,17 +805,56 @@ def with_index(self, index):
 
 
 def time_frame(
-    columns=None,
+    columns: Optional[Dict[str, Collection]] = None,
     *,
-    index=None,
-    start=None,
-    freq=None,
-    static=None,
-    tdims=None,
-    length=None,
-    default_tdim=-1,
-    metadata=None,
+    index: Optional[Periods] = None,
+    start: Optional[Union[Period, str]] = None,
+    freq: Optional[str] = None,
+    static: Optional[Dict[str, Any]] = None,
+    tdims: Optional[Dict[str, int]] = None,
+    length: Optional[int] = None,
+    default_tdim: int = -1,
+    metadata: Optional[Dict] = None,
 ):
+    """Create a `zebras.TimeFrame` object that represents one
```

Comment:
We should always use double backticks for sphinx when referencing types, etc:

```suggestion
    """Create a ``zebras.TimeFrame`` object that represents one
```